#! /bin/sh /usr/share/dpatch/dpatch-run
## 15_alpha_ftbfs_fix.dpatch by Brian Nelson <pyro@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Rename q_atomic_test_and_set_release_ptr to
## DP: q_atomic_test_and_set_ptr, as suggested by Isaac Clerencia
## DP: <isaac@debian.org>, to fix a FTBFS on alpha.

@DPATCH@
diff -urNad qt4-x11-4.1.0~/src/corelib/arch/alpha/arch/qatomic.h qt4-x11-4.1.0/src/corelib/arch/alpha/arch/qatomic.h
--- qt4-x11-4.1.0~/src/corelib/arch/alpha/arch/qatomic.h	2005-12-16 03:52:24.000000000 -0800
+++ qt4-x11-4.1.0/src/corelib/arch/alpha/arch/qatomic.h	2006-01-10 09:12:35.000000000 -0800
@@ -87,7 +87,7 @@
     return ret;
 }
 
-inline int q_atomic_test_and_set_release_ptr(volatile void *ptr, void *expected, void *newval)
+inline int q_atomic_test_and_set_ptr(volatile void *ptr, void *expected, void *newval)
 {
     register void *ret;
     asm volatile("1:\n"
